src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,20 +56,39 @@
   // private final XboxController m_controller = new XboxController(0);
   private Joystick joystick1;
   private Joystick joystick2;
+  private Joystick joystick3;
+
+    // joystick1 buttons
+  private JoystickButton driverIntakeButton;
+  private JoystickButton driverFeedButton;
+  private JoystickButton driverEjectButton;
+  private JoystickButton driverStartShootin;
 
   // joystick2 buttons
   private Button resetGyro;
   private JoystickButton startShootin;
   private JoystickButton stopShootin;
-
-  private JoystickButton intakeDeploy;
-  private JoystickButton intakeRetract;
-
-  private JoystickButton intakeButton;
-  private JoystickButton feedButton;
-  private JoystickButton ejectButton;
   private JoystickButton alignTarget;
 
+// joystick3 buttons
+  private JoystickButton climberArmTwoUpButton;
+  private JoystickButton climberArmOneUpButton;
+  private JoystickButton climberArmTwoDownButton;
+  private JoystickButton climberArmOneDownButton;
+  private JoystickButton climberUpButton;
+  private JoystickButton climberDownButton;
+  private JoystickButton climberArmTwoIn;
+  private JoystickButton climberArmTwoOut;
```

Comment:
arm2 in/out will be a switch so add a // @todo here that these will need to be combined eventually

Reply:
I'm just removing the arm2 in, since it never gets used